lib/elixir/lib/io.ex

```diff
@@ -233,15 +233,49 @@ defmodule IO do
   @doc """
   Inspects and writes the given `item` to the device.
 
+  It's important to note that it returns the given `item` unchanged.
+  This makes it possible to insert an `IO.inspect/2` call into
+  virtually any place of the code or in the middle of a pipeline
```

Comment:
It's not bad as is, but currently this does not read as idiomatic English, imo. :) Just figured this is some documentation people new to the language might see frequently, so it should be as good as possible.